QEfficient/cloud/infer.py

```diff
@@ -48,15 +49,15 @@ def main(
 ) -> None:
     qpc_base_dir_name = get_qpc_dir_name_infer(num_cores, mos, batch_size, prompt_len, ctx_len, mxfp6, mxint8, device_group)
     prompt: List[str] = check_batch_size_and_num_prompts(prompt, prompts_txt_file_path, batch_size)
-    tokenizer = load_hf_tokenizer(model_name=model_name, cache_dir=cache_dir, hf_token=hf_token)
+    tokenizer, model_path = load_hf_tokenizer(model_name=model_name, cache_dir=cache_dir, hf_token=hf_token, model_path=model_path)
```

Comment:
Why are we returning `model_path` again here?

Reply:
With `ignore_patterns` change which I did, all the model files were getting downloaded at this point. I wanted to keep uniformity in the code ahead and have `model_path` for both custom models and model downloaded from HF

If you think of it `model_path` = `cache_dir` + `model_name` 

Now that we will download only the tokenizer related files in this step, this change won't make sense as the model files will be downloaded in export